forem/forem
# Revert "Bump @storybook/builder-webpack5 from 6.3.12 to 6.4.1 (#15533)"

This reverts commit 90dcb5af475ad634e342d7ea3fdbf0ea7f8751bb.



## What type of PR is this? (check all applicable)


## Description

This reverts https://github.com/forem/forem/pull/15533, as although the CI build passed successfully, running `yarn storybook` locally was failing with error `ERR! Error: Cannot find module 'glob-base'`.

The issue appears to be with `@storybook/manager-webpack5`, but as mentioned in the original PR, this version bump to storybook requires all the dependencies to be bumped together, rather than addressed separately, which is why I'm just reverting the commit for now so we can unblock storybook for local development and look into this version upgrade separately.

## Related Tickets & Documents

Reverts https://github.com/forem/forem/pull/15533

## QA Instructions, Screenshots, Recordings

Run `yarn install && yarn storybook` locally and confirm storybook runs as expected

### UI accessibility concerns?

N/A

## Added/updated tests?

- [X] No, and this is why: dependency downgrade

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_
